src/main/java/apoc/load/Xml.java

```diff
@@ -27,31 +45,119 @@
     @Context public GraphDatabaseService db;
 
     @Procedure
-    @Description("apoc.load.xml('http://example.com/test.xml', false) YIELD value as doc CREATE (p:Person) SET p.name = doc.name load from XML URL (e.g. web-api) to import XML as single nested map with attributes and _type, _text and _childrenx fields.")
-    public Stream<MapResult> xml(@Name("url") String url, @Name(value = "simple", defaultValue = "false") boolean simpleMode) {
-        return xmlToMapResult(url, simpleMode);
+    @Description("apoc.load.xml('http://example.com/test.xml', 'xPath', false) YIELD value as doc CREATE (p:Person) SET p.name = doc.name load from XML URL (e.g. web-api) to import XML as single nested map with attributes and _type, _text and _childrenx fields.")
+    public Stream<MapResult> xml(@Name("url") String url, @Name(value = "path",defaultValue = "/") String path, @Name(value = "simple", defaultValue = "false") boolean simpleMode) throws Exception {
```

Comment:
For efficiency reasons doesn't make it sense to use the previous implementation that doesn't do XPath if there is none given / wanted ?